spec/models/generic_announcement_ability_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe GenericAnnouncement do
```

Comment:
Yes, in some places: https://github.com/Coursemology/coursemology2/blob/9f23d6a2eec4dd59b607f6c7b7e7d0a43cc88eda/app/controllers/concerns/application_announcements_concern.rb#L5